backend/src/test_calculator.py

```diff
@@ -8,7 +9,111 @@ def setUp(self):
         self.calculator = Calculator()
 
     def test_add(self):
-        self.assertEqual(self.calculator.addition(1, 2), 3)
+        self.assertEqual(self.calculator.addition(1, 2), 3, "Simple")
+        self.assertEqual(self.calculator.addition(-12, -32), -44, "Negative numbers")
+        self.assertEqual(self.calculator.addition(0, 32), 32, "1st number 0")
+        self.assertEqual(self.calculator.addition(33, 0), 33, "2nd number 0")
+        self.assertEqual(self.calculator.addition(33, -33), 0, "Result equal to 0")
+        self.assertRaises(TypeError, self.calculator.addition, None, None)
+        self.assertRaises(TypeError, self.calculator.addition, 'face', 5)
+        self.assertRaises(TypeError, self.calculator.addition, 5, 'face')
+
+    def test_substraction(self):
+        self.assertEqual(self.calculator.subtraction(10, 6), 4, "Simple")
+        self.assertEqual(self.calculator.subtraction(-10, -6), -4, "Both negative")
+        self.assertEqual(self.calculator.subtraction(10, -6), 16, "Second negative")
+        self.assertEqual(self.calculator.subtraction(-10, 6), -16, "First negative")
+        self.assertEqual(self.calculator.subtraction(22.23, 15.71), 6.52, "Float numbers")
+        self.assertEqual(self.calculator.subtraction(13, 0), 13, "Second number 0")
+        self.assertEqual(self.calculator.subtraction(0, 13), -13, "First number 0")
+        self.assertEqual(self.calculator.subtraction(129, 129), 0, "Result equal to 0")
+        self.assertRaises(TypeError, self.calculator.subtraction, None, None)
+        self.assertRaises(TypeError, self.calculator.subtraction, 'face', 5)
+        self.assertRaises(TypeError, self.calculator.subtraction, 5, 'face')
+
+    def test_multiplication(self):
+        self.assertEqual(self.calculator.multiplication(10, 6), 60, "Simple")
+        self.assertEqual(self.calculator.multiplication(-10, -6), 60, "Both negative")
+        self.assertEqual(self.calculator.multiplication(-10, 6), -60, "First negative")
+        self.assertEqual(self.calculator.multiplication(10, -6), -60, "Second negative")
+        self.assertEqual(self.calculator.multiplication(0, 24), 0, "First number 0")
+        self.assertEqual(self.calculator.multiplication(24, 0), 0, "Second number 0")
+        self.assertEqual(self.calculator.multiplication(24.23, 12.56), 304.3288, "Float numbers")
```

Comment:
Для проверки числел с плавающей запятой лучше использовать assertAlmostEqual
```suggestion
        self.assertAlmostEqual(self.calculator.multiplication(24.23, 12.56), 304.3288, "Float numbers")
```